### PR TITLE
Fix frontend CI

### PR DESCRIPTION
**Pull request by ivhus**

### CODE REVIEW

File: .github/workflows/docker_build.yml [hunk 1]
```diff
@@ -32,27 +32,34 @@ jobs:
...
     #   run: |
     #     echo $GITHUB_REF_NAME
     #     echo ${GITHUB_SHA::7}
 
-    - name: Build, tag, and push the image to Amazon ECR
+    - name: Build the image
       id: build-image
+      if: github.event_name == 'pull_request'
+      run: |
+        docker build -f Dockerfile .
```
Comment (vkomodey) [inline .github/workflows/docker_build.yml:52]: build phase for the human app should include build arguments:

```
REACT_APP_API_URL
REACT_APP_HCAPTCHA_SITE_KEY
REACT_APP_CIVIC_APP_ID
```

File: .github/workflows/docker_build.yml [hunk 2]
```diff
@@ -32,27 +32,34 @@ jobs:
...
+
+    - name: Build and push the image to Amazon ECR
+      id: build-push-image
       env:
         ECR_REGISTRY: ${{ steps.login-ecr.outputs.registry }}
         ECR_REPOSITORY: human-app-ui
-        IMAGE_TAG: "$GITHUB_REF_NAME-${GITHUB_SHA::7}"
+      if: github.event_name != 'pull_request'
```
Comment (vkomodey) [inline .github/workflows/docker_build.yml:59]: not sure I understand this. so if ANY event happened except pull request - we should trigger the build phase?

**Review by vkomodey — CHANGES_REQUESTED** [review]

File: .github/workflows/docker_build.yml (hunk 1 above)
Comment (ivhus) [reply .github/workflows/docker_build.yml:52]: build arguments added

File: .github/workflows/docker_build.yml (hunk 2 above)
Comment (ivhus) [reply .github/workflows/docker_build.yml]: This is necessary for docker push does not run on pull request (there are 2 steps in the pipeline with docker build command, but docker push is run only for the develop and master branches)

**Pull request merged**